staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3520,25 +3538,14 @@ func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, load
 		}
 	}
 
-	sgList := []string{}
-
-	for _, extraSG := range strings.Split(annotations[ServiceAnnotationLoadBalancerSecurityGroups], ",") {
-		extraSG = strings.TrimSpace(extraSG)
-		if len(extraSG) > 0 {
-			sgList = append(sgList, extraSG)
-		}
-	}
-
 	// If no Security Groups have been specified with the ServiceAnnotationLoadBalancerSecurityGroups annotation, we add the default one.
 	if len(sgList) == 0 {
 		sgList = append(sgList, securityGroupID)
 	}
 
-	for _, extraSG := range strings.Split(annotations[ServiceAnnotationLoadBalancerExtraSecurityGroups], ",") {
-		extraSG = strings.TrimSpace(extraSG)
-		if len(extraSG) > 0 {
-			sgList = append(sgList, extraSG)
-		}
+	extraSGList := getSGListFromAnnotation(annotations[ServiceAnnotationLoadBalancerExtraSecurityGroups])
```

Comment:
nit: can be replaced by `sgList = append(sgList, extraSGList...)`

Reply:
Yeah , I was looking at the Golang doc I couldn't figure out if we have something to just append without own loop.